module/People/src/People/Service/OrganizationCommandsListener.php

```diff
@@ -22,12 +22,23 @@ protected function onOrganizationCreated(StreamEvent $event) {
 	}
 	
 	protected function onOrganizationUpdated(StreamEvent $event) {
+		$id = $event->metadata()['aggregate_id'];
+		$entity = $this->entityManager->find(Organization::class, $id);
+		$entityUpdated = false;
```

Comment:
Questa variabile è tautologica: ti trovi in un metodo che risponde ad un evento di OrganizationUpdated, per cui è ovvio che l'organizzazione è stata aggiornata. Non ti importa sapere esattamente in cosa, ma di sicuro se è stato tirato un motivo ci sarà per cui non condizionare l'aggiornamento sul db a questa variabile